### PR TITLE
Set up CI script and upgrade node-sass

### PR DESCRIPTION
**Pull request by stephengeller**
## What does this change?

This PR brings in the CI behaviour that was previously hardcoded into TeamCity steps into a `scripts/ci.sh` script. It also upgraded the Node version to 10 from the now-deprecated 6.

Once this is merged in we can upgrade the deprecated dependencies more easily. 

## How to test

Clone the repository, checkout the branch, run `nvm use` to use node 10 and run `./scripts/ci.sh`. This should exit cleanly.

## How can we measure success?

The script passes in CI and the app still runs as intended.

### CODE REVIEW

**Comment by stephengeller** [comment]
You can have a look at this script passing in CI here: https://teamcity.gutools.co.uk/viewLog.html?buildId=553651&buildTypeId=EditorialTools_EditorialProductionMetrics

File: scripts/setup.sh [hunk 1]
```diff
@@ -13,6 +17,16 @@ if [ $? != "0" ]; then
...
+
+NODE_MAJOR_VERSION=$(node -v | cut -d "." -f 1)
+DESIRED_NODE_VERSION=$(cat "${DIR}/../.nvmrc")
+
+if [[ "${NODE_MAJOR_VERSION}" != *"${DESIRED_NODE_VERSION}"  ]]; then
+  echo -e "${red}Your node version ${NODE_MAJOR_VERSION}" does not match "${DESIRED_NODE_VERSION}"
+  echo -e "${red}Please run 'nvm use' to get the desired node version"
+  exit 1
+fi
+
```
Comment (stephengeller) [inline scripts/setup.sh:29]: This ensures you're running the correct version of node. This is also in the start script

File: scripts/ci.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
```
Comment (akash1810) [inline scripts/ci.sh:2]: I'd recommending a `set -e` to ensure fast failure. Without it we could upload a build to riffraff with a broken js build, for example.

Comment (stephengeller) [reply scripts/ci.sh:2]: Good shout! That's been added now.

File: .prettierrc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "tabWidth": 4
```
Comment (stephengeller) [inline .prettierrc:2]: A future PR can change this and prettify all the files

File: public/js/utils/history.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import createHistory from 'history/createBrowserHistory';
+import { createBrowserHistory as createHistory } from "history";
```
Comment (stephengeller) [inline public/js/utils/history.js:1]: `'history/createBrowserHistory'` is deprecated

**Comment by jennygrahamjones** [comment]
I'm encountering two errors running this locally, neither of which should be problems in CI: 

1. The 3 `controllers.MetricsDBSpec`Scala tests fail. I resolved this manually by deleting the database, but running the script twice will result in the test failures again. It looks like there needs to be some teardown of the database, for the tests to run/pass locally, otherwise the tests fail on the basis that they're trying to create duplicate DBs/insert duplicate metrics. 

1. The script errors out at: `[error] (root/*:riffRaffUpload) com.amazonaws.AmazonClientException: Unable to load AWS credentials from any provider in the chain`. If possible, we probably don't want the `riffRaffUpload` step to happen locally?
